plugins/MultiSites/vue/src/AllWebsitesDashboard/AllWebsitesDashboard.less

```diff
@@ -129,23 +129,16 @@
     align-items: flex-start;
     display: flex;
     justify-content: space-between;
-    margin: 0 40px 20px 40px;
+    margin: 0 0 20px;
     overflow-x: auto;
 
     .kpiCard {
+      display: grid;
       flex-basis: 0;
       flex-grow: 1;
       margin: 20px 0;
-      padding: 0 40px;
-      white-space: nowrap;
-
-      &:first-child {
-        padding-left: 0;
-      }
-
-      &:last-child {
-        padding-right: 0;
-      }
+      padding: 0;
+      justify-items: center;
```

Comment:
Have we already checked we can display those values fully centered?

Otherwise we could use a slowly decreasing `padding-left + padding-right`, e.g. start with `padding: 0 80px` and go down 20px per roughly 100px screenwidth until `padding: 0 20px`, and the same from 992px to 600px :thinking:

Reply:
I've tried to change that to a padding instead of centered content. There are now combinations of screen sizes and content where some of the lines will break. But that can't be solved much better, due to differing i18n content.